ShareTheTable/View Controllers/Profile Making/Add Photos/AddPhotosViewController.h

```diff
@@ -0,0 +1,27 @@
+//
+//  AddPhotosViewController.h
+//  ShareTheTable
+//
+//  Created by Andrea Gonzalez on 7/7/22.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol AddPhotosControllerDelegate
+@end
+
+@interface AddPhotosViewController : UIViewController
+@property (weak, nonatomic) IBOutlet UIButton *image5;
```

Comment:
Anytime you're creating a bunch of outlets for the same type of object, you can usually put them in a TableView or CollectionView (like in Flixter). In this case, it might be easier to make a CollectionView with an imageCell. This will make it easier to perfect the UI later on and avoid writing the same code multiple times, but what you have currently is mostly clean as well!

Reply:
Another way to make UI easier later is to use a stackView for your 5 images for even spacing.